packages/frontmatter/src/FrontmatterBlock.tsx

```diff
@@ -187,20 +188,29 @@ export function Journal({
   );
 }
 
+export type LaunchOptions = {
+  repo: string;
+  location: string;
+  binder?: string;
+  jupyterhub?: string;
+};
+
```

Comment:
I don't love this; I think instead `frontmatter` should be resolved according to https://mystmd.org/guide/frontmatter#table-frontmatter. But, for now, let's just get this in.